lib/ansible/modules/cloud/amazon/ec2_elb_lb.py

```diff
@@ -477,7 +477,12 @@ def __init__(self, module, name, listeners=None, purge_listeners=None,
         self.changed = False
         self.status = 'gone'
         self.elb_conn = self._get_elb_connection()
-        self.elb = self._get_elb()
+
+        try:
+            self.elb = self._get_elb()
+        except boto.exception.BotoServerError as e:
+            module.fail_json(msg=str(e))
```

Comment:
While fixing these, could you add a message and traceback? i.e. `module.fail_json(msg="Unable to get all load balancers", exception=traceback.format_exc(), **camel_dict_to_snake_dict(e.message))`
(traceback will need to be imported and camel_dict_to_snake_dict will need to be imported from ansible.module_utils.ec2).